### PR TITLE
notes: Generate changelog for v1.0.0-beta1 release

### PR DESCRIPTION
**Pull request by marclop**
## Description

Adds the changelog for the v1.0.0-beta1 release

## Types of Changes

- [x] Documentation

### CODE REVIEW

File: notes/v1.0.0-beta1.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
+# Changelog
+
+Release binaries can be downloaded with the following links:
+
+<https://download.elastic.co/downloads/ecctl/1.0.0-beta1/ecctl_1.0.0-beta1_darwin_amd64.tar.gz>
```
Comment (karencfv) [inline notes/v1.0.0-beta1.md]: The links need an extra newline between them so they show up in different lines (markdown is annoying 😄) 

Screen Shot 2019-12-23 at 10 45 32 AM

File: notes/v1.0.0-beta1.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+The command was broken since the UserID wasn't specified and the Key ID was being set instead of the User ID.
+The command now works as expected.
+
+## Docs
+<!-- Add the list of doc changes, remove otherwise.-->
+
+### Added ASCIIDoc for commands
+
+### Added auto-completion instructions to the docs
```
Comment (karencfv) [inline notes/v1.0.0-beta1.md]: This line isn't showing the header for some reason 🤔 could be good to use bold instead?

Screen Shot 2019-12-23 at 10 48 36 AM

File: notes/v1.0.0-beta1.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+### Detailed changes
+<!-- AUTOGENERATED COMMIT LIST -->
+
+97c3985 Adding ecctl icon to repo (#111)
```
Comment (karencfv) [inline notes/v1.0.0-beta1.md]: Same comment about the newlines. Unless this isn't as important, in which case leave as is :)

File: notes/v1.0.0-beta1.md (hunk 2 above)
Comment (marclop) [reply notes/v1.0.0-beta1.md]: It had a weird blank line there, it's fixed now

**Pull request merged**